### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
# Final
## In this branch I have done the followings:
- Added design to the pages.
- Added Functionalities to the app.
- Added Tests to check the renderings.

### CODE REVIEW

File: src/Redux/List.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      const findgame1Index = action.payload.results.findIndex(
+        (game) => game.id === 58175,
+      );
+      const findgame2Index = action.payload.results.findIndex(
+        (game) => game.id === 32,
+      );
+      state.items.splice(findgame1Index, 1);
+      state.items.splice(findgame2Index, 1);
+    }).addCase(fetchList.rejected, () => {
+      // console.log('rejected');
```
Comment (tufoinnkuo10) [inline src/Redux/List.js]: - [x] kindly remove commented code for clean code practice

File: README.md [hunk 2]
```diff
@@ -1,70 +1,202 @@
...
+- **[Details]** on click the user can see the detials of a certain game
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (tufoinnkuo10) [inline README.md:73]: - [x] please kindly ensure your project is deployed using Heroku, Netlify or GitHub Pages and a link added to readme. please check if it is working properly.
Screenshot (627)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Mhamad-Raad 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Design well followed
✅ Readme ok
✅ Test passed

-- video presentation was brief and straight to the point. all project parts were covered. great job.
Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**